content/docs/dvclive/ml-frameworks/pytorch-lightning.md

```diff
@@ -69,6 +82,52 @@ checkpointing at all as described in the
 
 ## Examples
 
+### Log model checkpoints
+
+Use `log_model` to save the checkpoints. DVCLive will first delete checkpoints
+from previous <abbr>experiments</abbr> (since DVC tracks checkpoints per
+experiment) and then <abbr>cache</abbr> the entire checkpoints directory using
+`Live.log_artifact()`. At the end of training, DVCLive will annotate the
+[`best_model_path`][`ModelCheckpoint`] with `type=model` and `name=best` for use
+in [Studio model registry].
```

Comment:
Need a good example of how to use it, but I think it's blocked by https://github.com/iterative/dvc/issues/9100#issuecomment-1644259684